x-pack/solutions/security/plugins/security_solution/public/configurations/routes.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { SecurityPageName } from '@kbn/security-solution-navigation';
+import type { SecuritySubPluginRoutes } from '../app/types';
+import { Configurations } from './page/configurations';
+import { PluginTemplateWrapper } from '../common/components/plugin_template_wrapper';
+import { SecurityRoutePageWrapper } from '../common/components/security_route_page_wrapper';
+import { CONFIGURATIONS_PATH } from '../../common/constants';
+
+const ConfigurationsPage = React.memo(function ConfigurationsPage() {
+  return (
+    <PluginTemplateWrapper>
+      <SecurityRoutePageWrapper pageName={SecurityPageName.configurations} redirectOnMissing>
```

Comment:
We should probably use `redirectIfUnauthorized` instead of `redirectOnMissing`. What happens when we access this page using _essentials_ tier?

Reply:
I checked on `Complete tier` and it redirected to `get_started` page. But I'll double test 👍

Reply:
That's odd, the link should not be missing but unauthorized. Let me know, if it works with `redirectOnMissing` I'll take a look.